server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -1058,6 +1064,38 @@ func (s *BuildBuddyServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// TODO(siggisim): Figure out why this JWT is overriding authority auth and remove.
 	ctx := context.WithValue(r.Context(), "x-buildbuddy-jwt", nil)
 
+	if lookup.Filename[0:2] == "//" {
+		// non-file targets
+		lookup.Filename = lookup.Filename[2:]
+		switch lookup.Filename {
+		case "buildlog":
+			c := chunkstore.New(
+				s.env.GetBlobstore(),
+				&chunkstore.ChunkstoreOptions{},
+			)
+			attempt := uint64(0)
+			if n, err := strconv.ParseUint(params.Get("attempt"), 10, 64); err == nil {
+				attempt = n
+			}
+			// Stream the file back to our client
+			w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=buildlog_for_invocation_%s", params.Get("invocation_id")))
+			w.Header().Set("Content-Type", "application/octet-stream")
+			io.Copy(
```

Comment:
io.Copy returns an error -- should we check it and either return or log if it's set?

Reply:
Yeah, sounds good; I spent a long time trying to use `http.ServeContent` (which returns nothing) here instead, but it was just serving nothing (still don't really understand why). So when I switched to `io.Copy`, I forgot to start doing error checking :|